README.md

```diff
@@ -75,6 +75,10 @@ Check out the [Payload website](https://payloadcms.com/docs/getting-started/what
 
 There are lots of good conversations and resources in our [GitHub Discussions board](https://github.com/payloadcms/payload/discussions). If you're struggling with something, chances are, someone's already solved what you're up against. Searching Discussions will often provide very helpful tips and tricks.
 
+##### Contributions
```

Comment:
I think this should go a bit higher. After `Documentation` as an H3 would be great.